ee/clickhouse/queries/funnels/funnel_unordered.py

```diff
@@ -0,0 +1,64 @@
+from itertools import combinations
+from typing import List
+
+from ee.clickhouse.queries.funnels.base import ClickhouseFunnelBase
+
+
+class ClickhouseFunnelUnordered(ClickhouseFunnelBase):
+    def get_query(self, format_properties):
+        return self.get_step_counts_query()
+
+    def get_step_counts_query(self):
+
+        max_steps = len(self._filter.entities)
+        union_queries = []
+        entities_to_use = list(self._filter.entities)
+
+        partition_select = self.get_partition_cols(1, max_steps)
+        sorting_condition = self.get_sorting_condition(max_steps)
+
+        for i in range(max_steps):
+            inner_query = f"""
+                SELECT 
+                person_id,
+                timestamp,
+                {partition_select}
+                FROM ({self._get_inner_event_query(entities_to_use, f"events_{i}")})
```

Comment:
Do we need to pass this all the way through since we already have access to filter.entities in the functions further down the chain?

Reply:
Sadly, yes, because this part switches the entities around, making `step_0` the next entity in the list, without touching `self.filter._entities`.

It seemed a lot more complicated to do this in the top level query generation, since that would've involved modifying every query generation bit, instead of just the entities list.

Reply:
Or more generally, what is the reason behind needing to pass the list of entities down if `_get_entity_query` is already handling all the entities in self._filter.entities

Reply:
got it